lib/src/base-styles/_nx-btn.scss

```diff
@@ -56,7 +58,7 @@
   @extend %nx-btn--disabled;
 
   background-color: $nx-blue-600;
-  border-color: $nx-blue-600;
+  border-color: transparent;
```

Comment:
This part of the primary button styles was wrong.  There should be no visible border according to zeplin